app/src/main/res/layout/fragment_upload_media_detail_fragment.xml

```diff
@@ -82,7 +82,7 @@
                     android:layout_marginTop="@dimen/standard_gap"
                     android:orientation="vertical">
 
-                    <fr.free.nrw.commons.widget.HeightLimitedRecyclerView
+                    <androidx.recyclerview.widget.RecyclerView
```

Comment:
@macgills This change should be in first commit or with HLRV commit?

Reply:
first commit, the important thing is we can easily get the HLRV back, the usage is easy to replace, the implementation less so